snuba_sdk/function.py

```diff
@@ -81,7 +88,8 @@ def validate(self) -> None:
                 )
             for param in self.parameters:
                 if not isinstance(
-                    param, (Column, CurriedFunction, Function, Identifier, Lambda)
+                    param,
+                    (Column, CurriedFunction, Function, Identifier, Lambda, Expression),
```

Comment:
I'm guessing you added `Expression` to this to support Functions of Timeseries. However I don't want to add the Expression class here. All of these other classes are subclasses of Expression, the idea is to be restrictive of which specific ones are allowed in Functions.

Reply:
Yes, that was my intention. I initially tried it with `Timeseries`, but that would make both modules require each other.

Reply:
Resolved with `Formula` class

Reply:
Shouldn't this be removed?